packages/syft/tests/syft/worker_test.py

```diff
@@ -222,15 +220,25 @@ def post_add(context: Any, name: str, new_result: Any) -> Any:
     action_object.syft_post_hooks__["__add__"] = []
 
 
-def test_worker_serde() -> None:
-    worker = Worker()
+def test_worker_serde(worker) -> None:
     ser = sy.serialize(worker, to_bytes=True)
     de = sy.deserialize(ser, from_bytes=True)
 
     assert de.signing_key == worker.signing_key
     assert de.id == worker.id
 
 
+@pytest.fixture(params=[0])
+def worker_with_proc(request):
```

Comment:
how is this different from the other worker fixture?

Reply:
This is a parameterized worker fixture that is made to configure the process count, only required for worker_test.